lib/rspec/core/option_parser.rb

```diff
@@ -70,7 +70,18 @@ def parser(options)
           bisect_and_exit(argument)
         end
 
-        parser.on('--[no-]fail-fast', 'Abort the run on first failure.') do |value|
+        parser.on('--[no-]fail-fast[=COUNT]', 'Abort the run after a certain number of failures (1 by default).') do |argument|
```

Comment:
I've added the "=" intentionally because if I drop it, the parser will believe that any other following string is a parameter.

ie
` --fail-fast -profile`

won't work because it will believe that `-profile` is the parameter for `--fail-fast`. Am I doing something wrong?